platform/darwin/src/MGLPolyline.mm

```diff
@@ -73,38 +73,34 @@ - (CLLocationCoordinate2D)coordinate {
                     return coordinates[i];
                 }
                 to = MGLRadianCoordinateFromLocationCoordinate(coordinates[i - 1]);
-                MGLRadianDirection direction = [self direction:from to:to] - 180;
+                CLLocationDirection direction = [self direction:from to:to] - 180;
                 MGLRadianCoordinate2D otherCoordinate = MGLRadianCoordinateAtDistanceFacingDirection(from,
-                                                                                                     overshoot/MGLMetersPerRadian,
+                                                                                                     overshoot/mbgl::util::M2PI * mbgl::util::EARTH_RADIUS_M,
```

Comment:
It was probably preferable to use `mbgl::util::EARTH_RADIUS_M` instead of `MGLMetersPerRadian`, as I mentioned in https://github.com/mapbox/mapbox-gl-native/pull/8713#discussion_r115836563. The two values are within the same order of magnitude, which is a good sign. 😉